src/main/java/domain/Crew.java

```diff
@@ -0,0 +1,121 @@
+package domain;
+
+import domain.constants.AttendanceStatus;
+import domain.constants.ExpulsionStatus;
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+public class Crew {
+    private static final LocalDate CHRISTMAS_DATE = LocalDate.of(2024, 12, 25);
+    private final CrewName name;
+    private final List<Attendance> attendances;
+
+    public Crew(final String name, final List<Attendance> attendances) {
+        this.name = new CrewName(name);
+        this.attendances = new ArrayList<>(attendances);
+    }
+
+    public static Crew of(final String name, final LocalDate today) {
+        final List<Attendance> attendances = IntStream.range(1, today.getDayOfMonth())
+                .mapToObj(today::withDayOfMonth)
+                .filter(Crew::canDateAttendance)
+                .map(Attendance::empty)
+                .toList();
+        return new Crew(name, attendances);
+    }
+
+    private static boolean canDateAttendance(final LocalDate today) {
+        if (today.getDayOfWeek() == DayOfWeek.SATURDAY || today.getDayOfWeek() == DayOfWeek.SUNDAY
+                || today.equals(CHRISTMAS_DATE)) {
+            return false;
+        }
+        return true;
+    }
```

Comment:
아래와 같이 변경해볼 수 있을 것 같아요. 😃

```suggestion
    private static boolean canDateAttendance(final LocalDate today) {
        return !(today.getDayOfWeek() == DayOfWeek.SATURDAY || today.getDayOfWeek() == DayOfWeek.SUNDAY
                && today.equals(CHRISTMAS_DATE);
```

Reply:
메서드 파라미터로 들어오는 날짜가 꼭 오늘일 필요가 있나요?  
설령 오늘일지라도 Crew 객체의 입장에서 메서드 파라미터로 오늘 날짜가 들어오는지를 알 필요는 없을 것 같아요. 😃